src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -1312,7 +1311,7 @@ export function registerTerminalActions() {
 			primary: KeyMod.CtrlCmd | KeyCode.KeyW,
 			win: { primary: KeyMod.CtrlCmd | KeyCode.F4, secondary: [KeyMod.CtrlCmd | KeyCode.KeyW] },
 			weight: KeybindingWeight.WorkbenchContrib,
-			when: ContextKeyExpr.and(TerminalContextKeys.focus, ResourceContextKey.Scheme.isEqualTo(Schemas.vscodeTerminal), TerminalContextKeys.editorFocus)
```

Comment:
Do you know why it doesn't match that? It's meant to as it's a terminal editor.

Reply:
when I focus run `inspect context keys` and focus a terminal, here are the results. See how `resourceScheme` is `file`

![Screenshot 2023-11-15 at 9 35 34 AM](https://github.com/microsoft/vscode/assets/29464607/aebf046d-7654-4752-9aaf-925c497d70ee)

Reply:
The question I'm curious about then is why this URI isn't coming through:

https://github.com/microsoft/vscode/blob/e2670a457fbba846c8adebc70bcaec674c949866/src/vs/workbench/contrib/terminal/browser/terminalEditorInput.ts#L122-L123

Provided it's actually still vscode-terminal like it used to be.

Reply:
logging here and looks like it gets set correctly 🤔 so not sure why it doesn't work

https://github.com/hamirmahal/vscode/blob/0edd8b42dbbcbb682e59b9d8425466e37daa6420/src/vs/workbench/common/contextkeys.ts#L245

https://github.com/microsoft/vscode/assets/29464607/60d9be78-7896-45c8-bd19-682331d3c298

Reply:
also worked when I created a terminal in the editor area directly

Reply:
Ah ok, this works just fine on mac. It does not work on windows, looking into it.